plaso/parsers/dsv_parser.py

```diff
@@ -59,6 +62,13 @@ def __init__(self, encoding=None):
     """
     super(DSVParser, self).__init__()
     self._encoding = encoding
+    if py2to3.PY_2:
+      self._end_of_line = b'\n'
```

Comment:
What about files with `\r\n` end of line?

Reply:
This is to match the current behavior. In dfvfs text file (https://github.com/log2timeline/dfvfs/blob/master/dfvfs/helpers/text_file.py#L19) and line_reader_file where end of line defaults to `\n`, and it wasn't overridden here: https://github.com/log2timeline/plaso/blob/master/plaso/parsers/dsv_parser.py#L154